### PR TITLE
Component open tickets

### PR DESCRIPTION
**Pull request by alaabadra**

### CODE REVIEW

**Comment by alaabadra** [comment]
ok ahmed

File: client/src/components/OpenTicket/TheReply/style.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.text_reply{
+    margin-left: -15px;
+}
+.reply{
+    margin-left: 295px;
+}
+.content_reply{
+    width: 631px;
+    height: 300px;
+    margin-top: 10px;
```
Comment (FarahZaqout) [inline client/src/components/OpenTicket/TheReply/style.css]: collapse the margins into one line.
margin: 10px 0 0 300px;
also, using fixed pixels for margins might lead to responsiveness issues. Keep that in mind.

File: client/src/components/OpenTicket/TheReply/style.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    margin-left:300px;
+    background: #ECECEC;
+    box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.25);
+    border-radius: 8px;
+}
+.contentText-area{
+    width: 546px;
+    height: 164px;
+    margin-left: 33px;
```
Comment (FarahZaqout) [inline client/src/components/OpenTicket/TheReply/style.css]: collapse the margins into one line

File: client/src/components/OpenTicket/TheReply/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+      </div>
+      <div className="icon2_reply">
+        <h4 className="name_email2">BB:</h4>
+        <h5 className="title_email2">bb.email@gmail.com</h5>
+        <i className="far fa-plus-square icon2" />
+      </div>
+      <textarea className="contentText-area">
+
+        Yeah I also thought so, I recommend you do it the other way that I suggessted.
```
Comment (tomduggan85) [inline client/src/components/OpenTicket/TheReply/index.jsx]: @alaabadra Do you want this text inside the textarea to be a placeholder?  You could use the placeholder attribute: `<textarea classname="contentText-area" placeholder="your placeholder">`

File: client/src/components/OpenTicket/TheReply/style.css (hunk 1 above)
Comment (alaabadra) [reply client/src/components/OpenTicket/TheReply/style.css]: i used only margin-top not margin-top and other margin so here not found collapse margin 
but , i now modify in many files in code css => collapse margin

File: client/src/components/OpenTicket/ContentSubject/index.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './style.css';
+
+const TickectSubject = () => (
+  <div className="content_ticket">
```
[the remark this answers is not in the document]
Comment (alaabadra) [reply client/src/components/OpenTicket/ContentSubject/index.jsx]: ok , now i used bem in all files css

**Pull request merged**